api/rpc/account/account.proto

```diff
@@ -173,7 +173,7 @@ message AccountRequest {
   string name = 1;
 }
 
-message AccountInfo {
+message Account {
```

Comment:
Hate to tell you this, but Account isn't needed here. @generalhenry already created an Account type for the service: https://github.com/appcelerator/amp/blob/v0.6.x/api/rpc/account/account.proto#L174